custom-files/run.env

```diff
@@ -22,7 +22,7 @@ DR_EVAL_DEBUG_REWARD=False
 DR_TRAIN_CHANGE_START_POSITION=True
 DR_TRAIN_ALTERNATE_DRIVING_DIRECTION=False
 DR_TRAIN_START_POSITION_OFFSET=0.0
-DR_TRAIN_ROUND_ROBIN_ADVANCE_DIST=0.05
+DR_TRAIN_ROUND_ROBIN_ADVANCE_DIST=0.1
```

Comment:
@MaikelGit the default behaviour in the AWS Console is 0.05 for this setting, as there's only one worker.  The default in DeepRacer for Cloud is two workers, so whilst I agree there's an issue to fix here that you've found I would ask that you set this back to 0.05 and set the number of episodes to 40, then we have the same behaviour as the console (moves 5% along the track per episode 20 times per iteration), albeit with twice as many workers.

Also could you please update the readme.md file in the custom-files folder to reflect the new defaults?  

Thanks.